### DISCUSSION TITLE
O que é ReactJs

### DISCUSSION THREAD

**Discussion by diegobisb** [dieegobs/exemplo#8] [Q&A] [answered]
O que é ReactJs

**Comment by dieegobs** [comment, marked as answer]
O React é uma biblioteca JavaScript de código aberto com foco em criar interfaces de usuário em páginas web. É mantido pelo Facebook, Instagram, outras empresas e uma comunidade de desenvolvedores individuais. É utilizado nos sites da Netflix, Imgur, Feedly, Airbnb, SeatGeek, HelloSign, Walmart e outros.
